stable/admin/README.md

```diff
@@ -171,6 +171,8 @@ The following tables list the configurable parameters for the `admin` option of
 | `replicas` | Number of NuoDB Admin replicas | `1` |
 | `lbPolicy` | Load balancer policy name | `nil` |
 | `lbQuery` | Load balancer query | `nil` |
+| `loadBalancerService.enabled` | Whether to deploy a Layer 4 cloud load balancer service for the admin layer | `false` |
```

Comment:
# The name `loadBalancer` is very overloaded. Can we call this `adminK8sService` or something that makes it clear that this has nothing to do with load-balancing of TEs?
# We currently rely on a round-robin among admins to distribute database processes among admin processes (each database process is connected to exactly one admin process). If we do not use K8s round-robin load-balancing, will all database processes be bound to a particular admin process?

Reply:
Sure I'll do that in the new PR

Reply:
On 2nd point, this is specifically for disabling/enabling the external L4 service, the internal headless service is still in place, so admin connection using that would still be round robin.